### ISSUE TITLE
[BUG] 3.4.0 fails to launch on macos 10.15

### ISSUE THREAD

**Issue by Julusian** [bitfocus/companion#3036]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

It fails with the error:
```
Error: dlopen(/Applications/Companion.app/Contents/Resources/node_modules/@napi-rs/canvas-darwin-x64/skia.darwin-x64.node, 1): Symbol not found: __ZTTNSt3__118basic_stringstreamIcNS_11char_traitsIcEENS_9allocatorIcEEEE
  Referenced from: /Applications/Companion.app/Contents/Resources/node_modules/@napi-rs/canvas-darwin-x64/skia.darwin-x64.node
  Expected in: /usr/lib/libc++.1.dylib
```

This suggests that the canvas library was compiled targetting a newer macos than was being used. 

Hopefully this can be lowered to 10.15 again, or if not then the launcher window should have its minimum macos check updated to reflect this change in requirements

Upstream bug report: https://github.com/Brooooooklyn/canvas/issues/896

### Environment (please complete the following information)

```markdown
- OS: macos 10.15
- Browser: N/A
- Companion Version: 3.4.0
```

**Comment by Julusian** [comment]
For anyone who finds this, could you give https://builds.julusian.dev/companion-builds/companion-mac-x64-skia-recompile.dmg a try?
You will need to hold alt, right click then click open on both the dmg and companion for it to launch as it is not fully signed.

If this runs, then I can tidy up the fix and get it into the builds. I don't have a mac that can run 10.15 to try it with myself, so I am hoping this will work

**Comment by Julusian** [comment]
could someone give the 1.9.1+381 (or later) build of satellite a test? 
I think I have gotten to the bottom of this issue, once it is confirmed that build works I can get it released and applied to companion too

**Comment by Julusian** [comment]
This is fixed in 3.4.1
